dynamic_rest/routers.py

```diff
@@ -180,6 +182,24 @@ def register_resource(self, viewset, namespace=None):
             'viewset': viewset
         }
 
+        # Make sure the resource name isn't registered, either
+        # TODO: Think of a better way to clean this up, there's a lot of
```

Comment:
resource_key is the DB table name which contains the app label, so technically it supports two models with the same name, as long as they're in different apps/have different table names. In practice, it's probably 1:1, or close enough that we probably won't lose much from saying it should be.